WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -120,7 +122,6 @@ - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     DDLogVerbose(@"didFinishLaunchingWithOptions state: %d", application.applicationState);
-    [self.window makeKeyAndVisible];
```

Comment:
Out of curiosity: Is this actually **not** needed? instantiating a UIWindow makes it Key and Visible by default?